pkg/sbom/generator/spdx/spdx.go

```diff
@@ -467,6 +468,12 @@ type ExternalDocumentRef struct {
 	SPDXDocument       string   `json:"spdxDocument"`
 }
 
+// Can also contain name, comment, seeAlso
+type LicensingInfo struct {
+	LicenseId          string    `json:"licenseId"`
```

Comment:
```suggestion
	LicenseID          string    `json:"licenseID"`
```

Reply:
@imjasonh 

lefthand side is internal; right hand side of the the spec is case insensitive, the tag is upper case https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/#101-license-identifier-field the example is camel-case Id, the sample json files in the tools-java https://github.com/spdx/tools-java/blob/d95178bbc5c96e9b2afac7b51f271e8ddfe7105e/testResources/SPDXJSONExample-v2.3-with-exception.spdx.json#L21 show it as camel case. 

May i please keep the right hand side as is? i.e. "licenseId" to ensure the output we produce matches the spdx sample files?

Reply:
Oh absolutely, sorry about my overzealous capitalization